Uchu.World/Packets/GameMessages/Server/AddBuffMessage.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using RakDotNet.IO;
+using Uchu.Core;
+
+namespace Uchu.World
+{
+    public struct AddBuffMessage
```

Comment:
May be missing `[ServerGameMessagePacketStruct]` like the other game messages. This tells the reader/writer to add header information.